firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/Candidate.kt

```diff
@@ -18,7 +18,14 @@ package com.google.firebase.vertexai.type
 
 import java.util.Calendar
 
-/** A response generated by the model. */
+/**
+ * A `Candidate` represents a single response generated by the model for a given request.
+ *
+ * @property content The actual content generated by the model.
+ * @property safetyRatings A list of [SafetyRating]s describing the generated content.
+ * @property citationMetadata Metadata about the sources used to generate this content.
+ * @property finishReason The reason the model stopped generating content, if it exist.
```

Comment:
May be good to note that reason this would be missing is a streaming response, rather than random omission (unless this is also the case).

Reply:
There have been instances of the model not including it in the response.